CHANGELOG.md

```diff
@@ -114,6 +114,8 @@ v0.34.0-rc.0 (2023-06-01)
 
 - Fix issue where updating some modules' config (e.g. `loki.process`) could lead to a panic. (@thampiotr) 
 
+- Fix issue where using exporters in modules failed due to not passing the in-memory address dialer. (@mattdurham) 
```

Comment:
This is in the -rc.0 section of the changelog, should it be put to unreleased/main?

Reply:
Yes, I was jumping the gun a bit.